code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -298,9 +298,11 @@ var/bomb_set
 			var/datum/game_mode/nuclear/GM = ticker.mode
 			var/obj/machinery/computer/shuttle_control/syndicate/syndie_location = locate(/obj/machinery/computer/shuttle_control/syndicate)
 			if(syndie_location)
-				GM.syndies_didnt_escape = (syndie_location.z > 1 ? 0 : 1)	//muskets will make me change this, but it will do for now
+				GM.syndies_didnt_escape = (syndie_location.z != map_datum.centcomm_z) //Muskets will make me change this, but it will do for now

```

Comment:
That's Centcomm. The only thing it changes is that Nuke Ops have to move back to their base to have the best possible ending